[2024-12-16T18:18:19.900+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [queued]>
[2024-12-16T18:18:19.980+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [queued]>
[2024-12-16T18:18:19.980+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-16T18:18:19.992+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_data> on 2024-12-15 20:00:00+00:00
[2024-12-16T18:18:19.996+0000] {standard_task_runner.py:57} INFO - Started process 3966 to run task
[2024-12-16T18:18:19.998+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'sick_leave_etl_dag', 'transform_data', 'scheduled__2024-12-15T20:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/sick_leave_dag.py', '--cfg-path', '/tmp/tmpxnj3wpem']
[2024-12-16T18:18:20.000+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask transform_data
[2024-12-16T18:18:20.014+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T18:18:20.046+0000] {task_command.py:415} INFO - Running <TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T18:18:20.117+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sick_leave_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T20:00:00+00:00'
[2024-12-16T18:18:20.137+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T18:18:20.148+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sick_leave_etl_dag, task_id=transform_data, execution_date=20241215T200000, start_date=20241216T181819, end_date=20241216T181820
[2024-12-16T18:18:20.171+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T18:18:20.188+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-16T19:04:12.664+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [queued]>
[2024-12-16T19:04:12.671+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [queued]>
[2024-12-16T19:04:12.672+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-16T19:04:12.685+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_data> on 2024-12-15 20:00:00+00:00
[2024-12-16T19:04:12.689+0000] {standard_task_runner.py:57} INFO - Started process 5121 to run task
[2024-12-16T19:04:12.692+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'sick_leave_etl_dag', 'transform_data', 'scheduled__2024-12-15T20:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/sick_leave_dag.py', '--cfg-path', '/tmp/tmpwg8z5gb0']
[2024-12-16T19:04:12.694+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask transform_data
[2024-12-16T19:04:12.709+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T19:04:12.743+0000] {task_command.py:415} INFO - Running <TaskInstance: sick_leave_etl_dag.transform_data scheduled__2024-12-15T20:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T19:04:12.822+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sick_leave_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T20:00:00+00:00'
[2024-12-16T19:04:12.836+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T19:04:12.848+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sick_leave_etl_dag, task_id=transform_data, execution_date=20241215T200000, start_date=20241216T190412, end_date=20241216T190412
[2024-12-16T19:04:12.862+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T19:04:12.881+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
